### PR TITLE
Adds status and sendStatus

### PR DESCRIPTION
**Pull request by DrPandemic**
It exposes [status](http://expressjs.com/en/api.html#res.status) and [sendStatus](http://expressjs.com/en/api.html#res.sendStatus).

I was trying to play with `[@@bs.send.pipe: t]` but I was unable to get it to work as I wanted. Currently,
```ocaml
Response.sendString (Response.status res 400) "Not parsable"
```
produces 
```javascript
return res.status(400).send("Not parsable");
```
Idealy I would like to be able to do something like that
```ocaml
res
  |> Response.status 400
  |> Response.sendString "Not parsable
```

I tried with `external status : int => t = "status" [@@bs.send.pipe: t];` but got some type errors with the `Response.sendString`. Do you have an idea how this could be fixed?

Anyway, even without the pipe support, I think simply exposing `status` and `sendStatus` might be useful.

### CODE REVIEW

**Comment by stroiman** [comment]
I placed the following code in my own project, which allows piping (be aware that I'm using Reason 3 syntax):

```
open Express;

module Response = {
  include Express.Response;
  [@bs.send.pipe : t] external status : int => t = "status";
  [@bs.send.pipe : t] external sendString : string => done_ = "send";

};

    Express.Middleware.from(
      (_req, res: Express.Response.t, _next) =>
        res |> Response.status(401) |> Response.sendString("")
    )
```

But I didn't create a pull request, because I change the signature of the types - so it wouldn't be compatible with existing code.

**Review by glennsl — CHANGES_REQUESTED** [review]
This will need to be rebased and updated to reason 3.

I've also added a few people with opinions to review this, but if there's no repsonse in the next few days, ping me and I'll merge it.

Also, sorry for the lack of response around here. I'm not a maintainer on this repo specifically (I don't know if anyone is, really), but I've lost all trust in GitHub being able to notify of issues and PRs on repos you own, let alone just being a collaborator.

**Comment by bassjacob** [comment]
Sorry for the delay, life got in the way a bit. Will take a look at this over the weekend. @DrPandemic if you'd like I can update it to R3 syntax and get it in, or if you'd like to I'll merge it once that's done.

**Comment by ncthbrt** [comment]
@DrPandemic sendStatus is available on the master branch. Will publish tomorrow

**Pull request closed without merging**
